src/arithmetic/bound_deducer.cc

```diff
@@ -68,6 +68,41 @@ std::vector<const Object*> GetPath(Expr target, Expr expr) {
   return v.path_;
 }
 
+class BoundRemover : public IRMutator {
+ public:
```

Comment:
NOTE, we are in progress of updating IRMutators to new style, would be great if we can directly change it here related https://github.com/apache/incubator-tvm/pull/4607